scripts/create_gitlab_snapshot.py

```diff
@@ -0,0 +1,105 @@
+import datetime
+import logging
+from time import (
+    sleep,
+)
+from typing import (
+    Mapping,
+)
+
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    JSON,
+    config,
+    require,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    require(config.terraform_component == 'gitlab',
+            "Select the 'gitlab' component ('dev.gitlab' or 'prod.gitlab', for example).")
+    instance = gitlab_instance_info()
+    shutdown_instance(instance)
```

Comment:
You need to shut down the instance to which the volume is attached, if any, not the instance carrying the label. They might be different, on purpose, or by mistake.